src/nft/components/common/Loading/LoadingSparkle.css.ts

```diff
@@ -12,19 +12,21 @@ const pathAnim = keyframes({
 const pathAnimCommonProps = {
   animationDirection: 'alternate',
   animationTimingFunction: 'linear',
-  animation: `0.5s infinite ${pathAnim}`,
 }
 
 export const path = style({
   selectors: {
     '&:nth-child(1)': {
+      animation: `0.5s infinite ${pathAnim}`,
```

Comment:
I'm assuming this is moved  from b/c it doesn't work otherwise?  you can declare these as a constant

Reply:
yes the animations need to be on the individual elements or else the stars blinked instead of a gradual twinkle